lib/main.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+#include <fstream>
+#include "./reader.h"
+
+namespace config_reader {
+    using namespace std;
+
+    std::string *read_config(std::string config_file_path) {
+
+        std::string count;
```

Comment:
If you make count an int, you could use it to total the amount of lines in the file you are reading in your while loop.